meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb

```diff
@@ -1,40 +1,35 @@
-SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "d0ca9d072012d8ac333537d9b922d9242e022efe"
+SUMMARY = "GENIVI HMI for GDP"
 LICENSE  = "MPL-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
 
 DEPENDS = "dbus-c++ systemd wayland-ivi-extension qtquick1 qtbase"
+RDEPENDS_${PN} += "qtsvg"
 
-RDEPENDS_${PN} += "qtbase qtsvg"
-
```

Comment:
Please put each logical change in it's own patch/commit with it's own commit log, no matter how small. This patch should only and only contain changes need to rename the binary (source files/recipe renames can & should also be part of it but then you need to remove 'binary' from commit log).